netconf/transport_ssh.go

```diff
@@ -117,6 +122,17 @@ func NewSSHSession(conn net.Conn, config *ssh.ClientConfig) (*Session, error) {
 	return NewSession(t), nil
 }
 
+// NewSSHSessionForExternalClient creates a new NETCONF session using an existing ssh.Client
+// initiated and managed externally.
+func NewSSHSessionForExternalClient(client *ssh.Client) (*Session, error) {
+	t, err := clientToTransport(client)
```

Comment:
Don't need a separate function for this.

Reply:
I accept. This seems redundant. May be I added it to resemble similarity with `connToTransport` fn.